UIKitDZ/UIKitDZ/TabBar/LinkTabBarController.swift

```diff
@@ -0,0 +1,53 @@
+// LinkTabBarController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Настройка панели табБара
+final class LinkTabBarController: UITabBarController {
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tabBar.itemPositioning = .centered
```

Comment:
```suggestion
        tabBar.itemPositioning = .centered
```
 тут только методы вызываем